From 864d875b573c10458ff352e022071a87a241d50c Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Tue, 17 Nov 2020 02:32:40 +0100
Subject: [PATCH] modules: update OpenWrt

7cbbab7246 mac80211: Fix brcmfmac compile on layerscape/armv8_64b
28a85b8c2b layerscape: update kernel patch to fix build
3a8cfabe0c kernel: Update kernel 4.9 to version 4.9.243
9cdc02be88 kernel: Update kernel 4.14 to version 4.14.206
5625f5bc36 uci: Backport security fixes
d94c59f7eb oxnas: fix qc_prep return in sata driver after kernel 4.14.200
ac56d25361 ar71xx: mikrotik: bypass id check in __rb_get_wlan_data()
5d01d05608 kernel: Update kernel 4.14 to version 4.14.202
edda06c7b4 kernel: Update kernel 4.9 to version 4.9.240
27677af27c firewall: options: fix parsing of boolean attributes

Build-tested: x86-64
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 85dba15a..f5651a3a 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='packages routing luci gluon'
 
 OPENWRT_REPO=https://git.openwrt.org/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-18.06
-OPENWRT_COMMIT=6d94a6eca41da34f4df845d9876be2acfbf5881f
+OPENWRT_COMMIT=7cbbab7246b43209a6d940f3a35c9f9a364a0572
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-18.06
-- 
2.17.1

